app/utils/FeatureToggle.scala

```diff
@@ -14,20 +14,38 @@
  * limitations under the License.
```

Comment:
Does this need to be uncommented and looked at / removed?

Reply:
done